modules/apps/jaggery/mdm/src/mdm/units/policy-create/public/js/policy-create.js

```diff
@@ -1897,10 +1897,74 @@ var showHideHelpText = function (addFormContainer) {
     }
 };
 
+function formatRepo (user) {
+    if (user.loading) {
+        return user.text
+    }
+    if (!user.username){
```

Comment:
Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158